pixeltable/exprs/column_ref.py

```diff
@@ -126,18 +128,46 @@ def default_column_name(self) -> Optional[str]:
     def _equals(self, other: ColumnRef) -> bool:
         return self.col == other.col and self.perform_validation == other.perform_validation
 
+    def _df(self) -> 'pxt.dataframe.DataFrame':
+        tbl = catalog.Catalog.get().tbls[self.col.tbl.id]
+        return tbl.select(self)
+
+    def collect(self) -> 'pxt.dataframe.DataFrameResultSet':
```

Comment:
Let's remove this, it's really dangerous. If you do frame_view.frame.collect() it'll blow up your memory. show() works just fine for exploration purposes.

I know you can also do this with frame_view.select(frame_view.frame).collect(), but that requires more deliberation.

Reply:
Good point, I wasn't really thinking about it when I added it. I've removed `collect()` (but left in `show()`, `head()`, and `tail()`, which are bounded).
<!-- Reviewable comment -OCGuKixBAMAhas-XnEv:bqwssi7 -->
<!-- Sent from Reviewable.io -->